app/v1/controllers/SubmitUkPropertyBsasController.scala

```diff
@@ -67,10 +68,14 @@ class SubmitUkPropertyBsasController @Inject()(val authService: EnrolmentsAuthSe
       val featureSwitch = FeatureSwitch(appConfig.featureSwitch)
 
       val rawData = SubmitUkPropertyBsasRawData(nino, bsasId, AnyContentAsJson(request.body))
+
       val result =
         for {
           parsedRequest <- EitherT.fromEither[Future](requestParser.parseRequest(rawData))
           response <- {
+            //Submit asynchronously to NRS
+            nrsService.submit(nino, request.body.as[SubmitUKPropertyBsasRequestBody])
```

Comment:
could these be taken from the parsedRequest instead? like `nrsService.submit(nino, parsedRequest.body)` or whatever it's called

Reply:
I'll look into it

Reply:
That seems to be fine,I'll push up the changes